### PR TITLE
mysql server has tags

### PR DESCRIPTION
**Pull request by abdullahaslam306**

### CODE REVIEW

File: plugins/azure/keyvaults/keyVaultHasTags.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'KeyVault Has Tags',
```
Comment (alphadev4) [inline plugins/azure/keyvaults/keyVaultHasTags.js:5]:
```suggestion
    title: 'Key Vault Has Tags',
```

File: plugins/azure/mysqlserver/mysqlServerHasTags.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'MySQL Server Has Tags',
+    category: 'MySQL Server',
+    domain: 'Databases',
+    description: 'Ensures that Azure MySQL Server Has Tags',
```
Comment (alphadev4) [inline plugins/azure/mysqlserver/mysqlServerHasTags.js:8]:
```suggestion
    description: 'Ensure that Azure MySQL servers have tags associated.',
```

File: plugins/azure/mysqlserver/mysqlServerHasTags.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'MySQL Server Has Tags',
+    category: 'MySQL Server',
+    domain: 'Databases',
+    description: 'Ensure that Azure MySQL servers have tags associated.',
+    more_info: 'Tags help you to group resources together that are related to or associated with each other. It is a best practice to tag cloud resources to better organize and gain visibility into their usage.',
+    recommended_action: 'Modify MySQL server and add tags',
```
Comment (alphadev4) [inline plugins/azure/mysqlserver/mysqlServerHasTags.js:10]:
```suggestion
    recommended_action: 'Modify MySQL server and add tags.',
```

File: plugins/azure/keyvaults/keyVaultHasTags.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
```
Comment (AkhtarAmir) [inline plugins/azure/keyvaults/keyVaultHasTags.js:3]: Can you remove this file?

Comment (AkhtarAmir) [inline plugins/azure/keyvaults/keyVaultHasTags.js:3]: Can you remove this file?

**Review by AkhtarAmir — CHANGES_REQUESTED** [review]

**Review by AkhtarAmir — APPROVED** [review]

**Pull request merged**